### ISSUE TITLE
iOS - Bookmark - VoiceOver User Not Able to Delete Bookmark

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Comply-Chain#98]
While using VoiceOver, the user is not able to delete a bookmark. VoiceOver won't even focus on the delete bookmark icon.

**Comment by jfc3-dol** [comment]
VoiceOver now reads "Remove Bookmark" but does not tell the user which bookmark it is they are removing.

**Comment by KyleLivingstonDOL** [comment]
@jfc3-dol The user cannot remove a bookmark without actually touching the corresponding remove book mark icon, which is next to the header. Are you stating that the voice over needs to read "remove bookmark ::voice over reads the bookmarked Step/Topic::"?

**Comment by jfc3-dol** [comment]
@KyleLivingstonDOL,

VoiceOver should read the following "Remove bookmark NAME OF BOOKMARKED ITEM" such as "Remove bookmark Step 3: Develop a Code of Conduct Communicating the Code".

The user needs to know what item/bookmark they are removing. The application should also tell the user that the action was done too.

**Comment by Abychalil** [comment]
Addressed this issue. Please retest and verify whether it meets the required 508 standards.
@jfc3-dol

**Comment by jfc3-dol** [comment]
@Abychalil,

VoiceOver does not tell users which item the "Remove Bookmark" button is for when they swipe to it. Letting the user know, for example, the following "Remove bookmark XYZ " where XYZ is the screen/page the user bookmarked.

VoiceOver does tell the user which item is removed from the "Undo" pop-up, though, and if you don't "Undo", the deleted bookmark the bookmark is removed.

**Comment by jfc3-dol** [comment]
@Abychalil,
I can remove the bookmark, but as mentioned above, on February 18th, the user does not know what bookmark is being removed because VoiceOver reads "Remove Bookmark button". The word "button" is read because that is what it was coded to be.

**Comment by jfc3-dol** [comment]
@harpreetsdev @pbhatt17 @Abychalil

A VoceOver user can close/remove the bookmark but not tell which one it is from the remove bookmark button.

So I'm going to close this issue and open another one to make sure VoiceOver notifies the user what item they are removing unless it's already in GitHub.
